src/cryptonote_core/service_node_list.cpp

```diff
@@ -107,7 +107,10 @@ namespace service_nodes
       }
     }
 
-    m_rollback_events.push_back(std::unique_ptr<rollback_event>(new prevent_rollback(current_height)));
```

Comment:
Please explain this change, I don't understand the purpose.

Reply:
Actually, because we are scanning from the beginning of the list now (and not from 30 days any more), then it is fine if we just remove this line entirely. No need to prevent rollbacks anymore.